ray-curator/ray_curator/stages/classifiers/aegis.py

```diff
@@ -0,0 +1,555 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
naming nit.. should these be `stages/text_models/base.py` + stages/text_models/classifiers/{aegis, domain}.py` + `stages/text_models/embedding_model.py`?

Given even SemDedup's embedding generation looks very similar to this (and it need not live inside semdedup but rather here)..